VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: PC CHASSIS


BO_ 32 ControlCommand: 8 PC
 SG_ TargetAngularSpeed : 7|16@0- (0.01,0) [-327.68|327.67] "rad/s" Vector__XXX
 SG_ TargetLinearSpeed : 23|16@0- (0.001,0) [-32.768|32.767] "m/s" Vector__XXX
 SG_ FaultLevel : 39|8@0+ (1,0) [0|255] "" Vector__XXX

BO_ 1409 FeedbackStatus: 8 CHASSIS
 SG_ HeaderByte0 : 7|8@0+ (1,0) [0|255] "hex" Vector__XXX
 SG_ HeaderByte1 : 15|8@0+ (1,0) [0|255] "hex" Vector__XXX
 SG_ LeftSpeed : 32|16@1- (0.1,0) [-3276.8|3276.7] "rpm" Vector__XXX
 SG_ RightSpeed : 48|16@1- (0.1,0) [-3276.8|3276.7] "rpm" Vector__XXX


CM_ SG_ 32 TargetAngularSpeed "Angular speed command, Big Endian (Motorola)";
CM_ SG_ 32 TargetLinearSpeed "Linear speed command, Big Endian (Motorola)";
CM_ SG_ 1409 LeftSpeed "Left wheel speed feedback, Little Endian (Intel)";
CM_ SG_ 1409 RightSpeed "Right wheel speed feedback, Little Endian (Intel)";